### PR TITLE
Try to fix adding existing shows, with unicode chars.

### LINKED ISSUE #5426: Existing TV Show folder names with accented characters or apostrophes cause exception
When adding existing shows, if the tv show folder name has accented characters or apostrophes ('), Medusa does not add the show and an exception is shown in the log.

Branch: master 
Commit: 4614efc77151ded92ef458a09dec39f8bd5acfc6 
Version: v0.2.10 
Database: 44.12
--
Python Version: | 2.7.15 (v2.7.15:ca079a3ea3, Apr 30 2018, 16:30:26) [MSC v.1500 64 bit (AMD64)]
OS: | Windows-10-10.0.18252
Locale: | en_US.cp1252

Fresh installation of Medusa under Windows 10 1809.  I have lots of shows with accented characters in the name (I have a lot of Spanish and Latin American shows), as well as shows with ' in the folder name.

When trying to add "Dirk Gently's Holistic Detective Agency", the following appears in the log and the show is not added:
```
2018-10-16 10:58:19 INFO     SHOWQUEUE-ADD :: [4614efc] Starting to add show by ShowDir: U:\Dirk Gentlyâ€™s Holistic Detective Agency
2018-10-16 10:58:19 INFO     SHOWQUEUE-ADD :: [4614efc] TVDBv2: {u'session': <medusa.session.core.MedusaSession object at 0x00000000086AFF98>, u'cache': u'C:\\pyMedusa\\cache\\indexers\\TVDBv2', u'language': u'en', u'use_zip': True}
2018-10-16 10:58:22 INFO     SHOWQUEUE-ADD :: [4614efc] 312505: Unable to find the show in the database
Traceback (most recent call last):
  File "C:\pyMedusa\medusa\show_queue.py", line 521, in run
    self.show.location = self.showDir
  File "C:\pyMedusa\medusa\tv\base.py", line 57, in __setattr__
    super(TV, self).__setattr__(key, value)
  File "C:\pyMedusa\medusa\tv\series.py", line 398, in location
    raise ShowDirectoryNotFoundException(u'Invalid show folder!')
ShowDirectoryNotFoundException: Invalid show folder!
2018-10-16 10:58:24 ERROR    SHOWQUEUE-ADD :: [4614efc] Error trying to add show: Invalid show folder!
2018-10-16 10:58:25 INFO     SHOWQUEUE-REMOVE-SHOW :: [4614efc] 312505: Removing Dirk Gently's Holistic Detective Agency
```

The same happens when trying to add tv shows with accented characters in the name, such as "Diablo Guardián".

EDIT: Medusa added correctly the show "James Cameron's Story of Science Fiction", so I am not sure what is going on with '  in the folder name.

### PR DESCRIPTION
**Pull request by p0psicles**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

fix #5426 #5661 #5618

### CODE REVIEW

File: themes-default/slim/views/addShows_addExistingShow.mako [hunk 1]
```diff
@@ -171,7 +172,7 @@ window.app = new Vue({
                 const seriesToAdd = [dir.selectedIndexer, dir.path, seriesId, dir.metadata.seriesName]
                     .filter(i => typeof(i) === 'number' || Boolean(i)).join('|');
 
-                formData.append('shows_to_add', encodeURIComponent(seriesToAdd));
+                formData.append('shows_to_add', seriesToAdd);
```
Comment (p0psicles) [inline themes-default/slim/views/addShows_addExistingShow.mako]: @OmgImAlexis is this okay?

File: medusa/server/web/home/add_shows.py [hunk 2]
```diff
@@ -532,7 +531,7 @@ def addExistingShows(self, shows_to_add=None, promptForSettings=None):
         elif not isinstance(shows_to_add, list):
             shows_to_add = [shows_to_add]
 
-        shows_to_add = [unquote_plus(x) for x in shows_to_add]
+        shows_to_add = [text_type(x, 'utf-8') if not isinstance(x, text_type) else x for x in shows_to_add]
```
Comment (p0psicles) [inline medusa/server/web/home/add_shows.py]: @medariox is this okay?

Comment (medariox) [reply medusa/server/web/home/add_shows.py]: Yes 👍

File: themes-default/slim/views/addShows_addExistingShow.mako [hunk 3]
```diff
@@ -163,6 +163,7 @@ window.app = new Vue({
             formData.append('promptForSettings', this.promptForSettings);
             dirList.forEach(dir => {
                 const originalIndexer = dir.metadata.indexer;
+                const { quoteUrl } = this;
```
Comment (medariox) [inline themes-default/slim/views/addShows_addExistingShow.mako]: What is the `quoteUrl` for?

File: themes-default/slim/views/addShows_addExistingShow.mako (hunk 1 above)
Comment (sharkykh) [reply themes-default/slim/views/addShows_addExistingShow.mako]: tl;dr: I think it should be okay, as long as you're consistent (removing the "decode" on the receiving end)

IIRC I just left it as it was on the old code, which _instead_ of sending a `POST` request with all the data, _it created_ a URL to navigate to (`GET` with query params).

**Original:** https://github.com/pymedusa/Medusa/commit/c408a6a0e0b65add222a8ebd2e7512f8a1958f21#diff-03bf01eea7b4fadd7983b2e8bb1f43daR108
**Switch to `POST`**: https://github.com/pymedusa/Medusa/commit/0e0449196eeef5138405dad9199f63cb279f2d97

**Comment by medariox** [comment]
This also fixes https://github.com/pymedusa/Medusa/issues/5360

**Pull request merged**